src/reducers/index.js

```diff
@@ -209,6 +212,18 @@ const slice = createSlice({
       state.scrolling = true;
       state.scrollX = scrollX;
     },
+    updateRowHeights(state) {
```

Comment:
how about we pass an optional parameter updatedRowIndex, this way we wont need to set rowUntilOffsetsAreExact as 0 everytime. 
For eg lets say there are 10000 rows and we have updated the height of 5000th row in that case we dont need to invalidate row heights and offsets of rows before 5000th row and we can simply set 
`state.getInternal().rowUntilOffsetsAreExact = updatedRowIndex //optional param` and if this param is not passed then we can set it to 0
Furthermore if the current  rowUntilOffsetsAreExact is less than the updatedRowIndex we dont need to change state (rowUntilOffsetsAreExact)

Reply:
I think is ok. I have added this parameter